docs/usage.md

```diff
@@ -118,11 +118,121 @@ directory.
 
 This means that analysis of these files failed and there is no Clang Static Analyzer output for these compilation commands.
 
-### Showing new bugs in your code after local edit
+
+## Step5: Integrate CodeChecker into your CI loop
+
+This section describes a recommended "way of working" about how CodeChecker is designed to be used
+in a CI environment to
+* generate daily report summaries
+* implement CI guard to prevent the introduction of new bugs into the code-base.
+
+In CodeChecker each bug has a unique hash identifier that is independent of the exact line of code 
```

Comment:
Should we highlight here that the hash is independent of the exact line of the code?
Actually it is independent of the line number if a comment is added to the source line the hash will change.